packages/frontend/preview/src/lib/lpc/lpc_preview_view_model.svelte.ts

```
@@ -13,30 +16,11 @@ import {
-} from '$lib/data/lpc_url_config';
-import { routerService } from '$services';
-import type { Application } from './lpc_pixi_facade';
-import { Container, Graphics, Sprite, Texture } from './lpc_pixi_facade';
-
-// Use the generated catalog directly — all slots with verified webp assets.
-const _getSlots = () => getLpcCatalog().slots;
+import type { AssetResolver } from '@aikami/types';
+import { Container, Graphics, Rectangle, Sprite, Texture } from 'pixi.js';
+import { createLpcRenderer, detectLpcSheetLayout, getLpcSpriteAnchor } from './lpc_renderer';
+import type { LpcRenderer } from './lpc_renderer';
+import { decodeLpcPreviewState, encodeLpcPreviewState, type LpcPreviewState } from './preview_url_state';
```

Biome (2.5.7)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)